ryot_compass/src/pancam/mod.rs

```diff
@@ -0,0 +1,378 @@
+use bevy::{math::vec2, prelude::*, render::camera::CameraProjection, window::PrimaryWindow};
```

Comment:
I know we only use for compass, but I wonder if we want to make it part of Ryot.